packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -83,6 +83,8 @@ abstract class FlutterCommand extends Command {
       help: 'Build a release version of your app${defaultToRelease ? ' (default mode)' : ''}.');
   }
 
+  void set defaultBuildMode(BuildMode buildMode) => _defaultBuildMode = buildMode;
```

Comment:
Let's use the full function syntax here (not the function shorthand) since we're not returning a value.